seves/settings.py

```diff
@@ -335,3 +335,5 @@
 
 if DEBUG:
     DSFR_USE_INTEGRITY_CHECKSUMS = env.bool("DSFR_USE_INTEGRITY_CHECKSUMS", True)
+
+SEND_NOTIFICATIONS = env("SEND_NOTIFICATIONS", default=True)
```

Comment:
Je viens de vérifier ici, si tu fais juste `env()`, c'est pas casté en booléen donc si tu mets `SEND_NOTIFICATIONS = False` dans le dotenv, tu vas récupérer la chaîne non-vide `"False"` et donc `settings.SEND_NOTIFICATIONS is False`  ne devrait jamais être vraie.

*Edit* : ah nan, pardon le code déduit le cast du type de `default`. J'ai rien dit.